conf/ejabberd.yml.tpl

```diff
@@ -434,9 +454,6 @@ modules:
 ###   ============
 ###   HOST CONFIG
 
-certfiles:
-  - "/opt/ejabberd/ssl/*.pem"
```

Comment:
This is duplicate, maybe we need another option that toggles this when env vars are not set?